codegen.yml

```diff
@@ -0,0 +1,21 @@
+overwrite: true
+schema: 'https://alpha-stts.tekit.io/graphql'
```

Comment:
@pugakn can we pull this from the environment variable?/

Reply:
@pugakn  Config can also be a typescript file. I've pushed the commit to replace codegen.yml with codegen.ts